### PR TITLE
To/16 Changed Text

### PR DESCRIPTION
**Pull request by dfuerk96**
Changed Text.

### CODE REVIEW

File: Rollenwechsel.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    <title>HORST-Rollenwechsel</title>
+  <link rel="stylesheet" href="stylesheet.css" type="text/css">
+  </head>
+  <body>
+    <div>HORST</div>
+
+    <li><a href="Grundlagen.html">Grundlagen</a></li>
+    <li><a href="Bedienung.html">Bedienung</a></li>
+    <li><a href="Bargeldwechsel.html">Bargeldwechsel</a></li>
+    <li><a href="Rollenwechsel.html">Rollenwechsel</a></li>
```
Comment (tobiasr97) [inline Rollenwechsel.html]: Why did you change the name?

Comment (dfuerk96) [reply Rollenwechsel.html]: This name made more sense.
